app/actions/websocket/posts.ts

```diff
@@ -223,7 +223,15 @@ export async function handlePostEdited(serverUrl: string, msg: WebSocketMessage)
         post.create_at = oldPost.createAt;
     }
 
-    if (oldPost.isPinned !== post.is_pinned) {
+    const oldFileIds = oldPost.metadata?.files?.map((f) => f.id) || [];
```

Comment:
Should we add tests for this file?

Reply:
As code is today, will add it in next PR.